src/api/types/ApiResponses.ts

```diff
@@ -0,0 +1,151 @@
+export interface PaymentRequestPageResponse extends PageResponse<PaymentRequest> {}
+

```

Comment:
Loved this! I believe this'd be the place to the `zod` types if we decide to move in that way